src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue

```diff
@@ -67,15 +67,18 @@ export default class ConsentContinuationOut extends Mixins(CountriesProvincesMix
     if (expiry) {
       return DateUtilities.apiToPacificDateTime(expiry, true)
     }
-    return '[unknown]'
+    return null
   }
 
   /** Check if Consent is Expired. */
   get isConsentExpired (): boolean {
-    const date = DateUtilities.apiToDate(this.filing.data?.consentContinuationOut?.expiry)
-    const daysToExpire = DateUtilities.daysBetweenTwoDates(new Date(), date)
-    if (isNaN(daysToExpire) || daysToExpire < 0) {
-      return true
```

Comment:
The bug was that `expiry` was empty, so `daysToExpire` was NaN and line 78 returned True.

I fixed this by looking at the situation a different way: In the template (lines 10 and 16), we only display one of the messages if expiry isn't empty. Also, I added a check for non-empty expiry in isConsentExpired() but, theoretically, this computed will never be used because expiry is checked first (short-circuit evaluation).

I was able to delete isFilingComplete() because we no longer care about this logic -- the template displays the expected HTML in all cases.